### DISCUSSION TITLE
JavaScript doesn't load

### DISCUSSION THREAD

**Discussion by Sasha99981** [PlaceholderAPI/PlaceholderAPI#755] [(2) Support] [answered]
Здраствуйте!
Я ввожу команду /papi ecloud download Javascript а потом /papi reload заклядываю папку и Джава скрипт не установлен в Плагине PlacehoderAPI.
Что мне с этим делать?

> **Translation from Google Translate (Edit by Andre_601)**
> Hello!
> I enter the command / papi ecloud download Javascript and then / papi reload I put the folder and the Java script is not installed in the PlacehoderAPI Plugin.
> What should I do about it?

**Comment by Andre601** [comment, marked as answer]
I've gone ahead and translated your message.
Please write in English next time, so that everyone can understand it, thanks.

**Reply by Sasha99981** [reply]
![image](https://user-images.githubusercontent.com/74203348/143937038-13b1e2fa-dd2f-497a-b6f0-f82dd9126fd9.png)
That's what I have after the team as it was and remained.

**Reply by Sasha99981** [reply]
And here is the folder where a
![image](https://user-images.githubusercontent.com/74203348/143937249-86de1ec6-edb9-4c53-9b5d-d901666cd77b.png)
ll the installers are

**Reply by Hepka** [reply]
> And here is the folder where a ![image](https://user-images.githubusercontent.com/74203348/143937249-86de1ec6-edb9-4c53-9b5d-d901666cd77b.png) ll the installers are

1. No use Hosting-Minecraft
2. JavaScript on PAPI runs only on Java 14 and below. Maybe that's why.

**Reply by Sasha99981** [reply]
Probably because of the hosting. Thanks, I'll look and tell you the result!
